python/lib/scheduler/dmod/scheduler/scheduler.py

```diff
@@ -345,6 +355,8 @@ def _ds_names_helper(cls, job: 'Job', worker_index: int, category: DataCategory,
             msg = "Attempting to start {} job {} with more than allowed max of {} required {} datasets."
             raise RuntimeError(msg.format(job.model_request.__class__.__name__, job.job_id, max_count, category))
         elif None in dataset_names is None:
```

Comment:
This is a bizarre condition. I don't know what it's supposed to do, but it's not gonna do it.

Reply:
Yeah, I think this is always `False`?